k8s/containers/seed-update/changelogs/base/base.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
```

Comment:
To make things simpler - I have just removed the seed-update.  The separate pull request for dbinit should have the required changes.